benchmark/agents/common.py

```diff
@@ -92,7 +92,7 @@ def proximity_detection(OGM):
 
 def heading_to_degree(heading):
     # +y = 0 rad. Note the 0 means up direction
-    return np.degrees((heading + math.pi) % (2 * math.pi))
+    return np.degrees(heading % (2 * math.pi))
```

Comment:
Just to be clear: you want the result of the heading to degrees to also be in the `+y = 0` rather than `+x = 0` direction? If the rest of the code operates on that assumption it is OK but I think that might also be confusing.

Reply:
yes, i do the calculation based on +y = 0 and clockwise increase to 360 degree. The old heading is degree in this way, so when i do migration, i just keep this way.